src/main/java/com/_up/megastore/controllers/interfaces/IUserController.java

```diff
@@ -15,4 +13,10 @@ public interface IUserController {
   void activateUser(@PathVariable UUID userId,
       @RequestBody @Valid ActivateUserRequest activateUserRequest);
 
+  @PostMapping("/recover-password/send-email")
+  void sendEmailToRecoverPassword(@RequestParam String email);
+
+  @PostMapping("/{userId}/recover-password")
+  void recoverPassword(@PathVariable UUID userId, @RequestParam String newPassword, @RequestParam UUID recoverPasswordToken);
```

Comment:
Estos al no devolver nada, deberian tener un response status 204 NO_CONTENT

```suggestion
  @PostMapping("/recover-password/send-email")
  @ResponseStatus(HttpStatus.NO_CONTENT)
  void sendEmailToRecoverPassword(@RequestParam String email);

  @PostMapping("/{userId}/recover-password")
  @ResponseStatus(HttpStatus.NO_CONTENT)
  void recoverPassword(@PathVariable UUID userId, @RequestParam String newPassword, @RequestParam UUID recoverPasswordToken);
```